maps/southern_cross/southern_cross_jobs.dm

```diff
@@ -113,10 +118,12 @@ var/const/access_explorer = 43
 	faction = "Station"
 	total_positions = 2
 	spawn_positions = 2
-	supervisors = "the chief medical officer"
+	supervisors = "the Chief Medical Officer"
 	selection_color = "#515151"
 	idtype = /obj/item/weapon/card/id/medical
 	economic_modifier = 4
 	access = list(access_medical, access_medical_equip, access_morgue, access_surgery, access_chemistry, access_virology, access_eva, access_maint_tunnels, access_external_airlocks, access_psychiatrist, access_explorer)
 	minimal_access = list(access_medical, access_medical_equip, access_morgue, access_explorer)
-	outfit_type = /decl/hierarchy/outfit/job/medical/sar
\ No newline at end of file
+
+	outfit_type = /decl/hierarchy/outfit/job/medical/sar
+	job_description = "A Search and Rescue recovers individuals who are injured or dead on the surface of Sif."
```

Comment:
"A Search and Rescue" seems kinda weird phrasing but idk if there's a proper term for 'someone who does SAR stuff'; maybe "Search and Rescue teams recover individuals..." ?

Reply:
`Search and Rescue is tasked with recovering [...]` ? Passive voice isn't the best but it dodges the problem here

Reply:
The tense of every other job description relies on it being more of a noun used to refer to an individual, though. SAR is almost always a plural word for a group overall.

Reply:
A Search and Rescue 'operator' may fix this.